src/shared/components/HeaderTextField/HeaderTextField.style.tsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled'
+import { fluidRange } from 'polished'
+import { Text } from '@/shared/components'
+import { colors, sizes, typography } from '@/shared/theme'
+
+export const Title = styled(Text)`
+  line-height: 1;
+  padding: ${sizes(1)} ${sizes(2)} ${sizes(2)};
+  ${fluidRange({ prop: 'fontSize', fromSize: '32px', toSize: '40px' })};
+  background-color: ${colors.gray[800]};
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
+  width: 100%;
+  max-width: 600px;
+  transition: 0.2s;
+  cursor: pointer;
+  &:hover {
+    filter: brightness(80%);
```

Comment:
I don't think you need a filter here, `opacity` should also work.

Reply:
On the second thought, I could be wrong. There will be custom background underneath this title, so, yeah, the filter may be better here.